package-lock.json

```diff
@@ -0,0 +1,9319 @@
+{
```

Comment:
can you just remove this file? I try to use yarn when possible :)

Reply:
done! :)